app/services/doc_auth_router.rb

```diff
@@ -201,26 +201,13 @@ def self.client(vendor_discriminator: nil, warn_notifier: nil, analytics: nil)
   # rubocop:enable Layout/LineLength
 
   def self.doc_auth_vendor(discriminator: nil, analytics: nil)
-    if IdentityConfig.store.doc_auth_vendor_randomize
-      target_percent = IdentityConfig.store.doc_auth_vendor_randomize_percent
-
-      if randomize?(target_percent, discriminator, analytics)
-        return IdentityConfig.store.doc_auth_vendor_randomize_alternate_vendor
-      end
-    end
-
-    IdentityConfig.store.doc_auth_vendor
-  end
+    case AbTests::DOC_AUTH_VENDOR.bucket(discriminator)
+    when :alternate_vendor
+      IdentityConfig.store.doc_auth_vendor_randomize_alternate_vendor
+    else
+      analytics&.idv_doc_auth_randomizer_defaulted if discriminator.blank?
 
-  def self.randomize?(target_percent, discriminator, analytics)
-    if discriminator.blank?
-      analytics&.idv_doc_auth_randomizer_defaulted
-      return false
+      IdentityConfig.store.doc_auth_vendor
     end
-
-    max_sha = (16 ** 64) - 1
-    user_value = Digest::SHA256.hexdigest(discriminator).to_i(16).to_f / max_sha * 100
```

Comment:
Since the `AbTestBucket` code injects the experiment name into the `percent` function, technically this will change the buckets from what is currently deployed.

However, since these are per-session UUID, the users are getting a free new chance every time they proof, so it's not like the same users should be getting the exact same behavior, so this shouldn't affect users significantly